### ISSUE TITLE
VerboseReporter fails when running tests

### ISSUE THREAD

**Issue by andrewhassan** [jtenner/as-pect#273]
### Summary 

If I set up `VerboseReporter` in my as-pect configuration file and run a test, it throws an error when trying to write to stdout.

My test simply contains a single `log("Hello world");` statement. as-pect is being run with the command `asp --config test/as-pect.config.js --summary` and the configuration described below.

### Configuration
```
const { VerboseReporter } = require("@as-pect/core");

module.exports = {
  include: ["**/*.spec.ts"],
  add: ["**/*.include.ts"],
  flags: {
    "--runtime": ["stub"],
    "--lib": ["node_modules"]
  },
  disclude: [/node_modules/],
  imports: {},
  performance: {
    enabled: false,
    maxSamples: 10000,
    maxTestRunTime: 5000,
    reportMedian: true,
    reportAverage: true,
    reportStandardDeviation: false,
    reportMax: false,
    reportMin: false
  },
  outputBinary: false,
  reporter: new VerboseReporter()
};

```

### Error
```
┃   [TestFile.ts] node_modules/@as-pect/assembly/assembly/index.ts --runtime stub --lib node_modules --validate --debug --binaryFile output.wasm --explicitStart --use ASC_RTRACE=1 --exportTable --importMemory --transform /Users/andrew/src/myproject/node_modules/@as-pect/core/lib/transform/index.js
┃
┃ /Users/andrew/src/myproject/node_modules/@as-pect/core/lib/reporter/VerboseReporter.js:74
┃             this.stdout.write(chalk(templateObject_1 || (templateObject_1 = __makeTemplateObject(["[Describe]: ", "\n\n"], ["[Describe]: ", "\\n\\n"])), group.name));
┃                         ^
┃
┃ TypeError: Cannot read property 'write' of null
┃     at VerboseReporter.onGroupStart (/Users/andrew/src/myproject/node_modules/@as-pect/core/lib/reporter/VerboseReporter.js:74:25)
┃     at VerboseReporter.onEnter (/Users/andrew/src/myproject/node_modules/@as-pect/core/lib/reporter/VerboseReporter.js:47:18)
┃     at TestContext.visit (/Users/andrew/src/myproject/node_modules/@as-pect/core/lib/test/TestContext.js:278:27)
┃     at TestContext.visit (/Users/andrew/src/myproject/node_modules/@as-pect/core/lib/test/TestContext.js:315:18)
┃     at TestContext.run (/Users/andrew/src/myproject/node_modules/@as-pect/core/lib/test/TestContext.js:201:14)
┃     at runBinary (/Users/andrew/src/myproject/node_modules/@as-pect/cli/lib/run.js:429:24)
┃     at /Users/andrew/src/myproject/node_modules/@as-pect/cli/lib/run.js:626:42
┃     at Object.main (/Users/andrew/src/myproject/node_modules/assemblyscript/cli/asc.js:998:10)
┃     at /Users/andrew/src/myproject/node_modules/@as-pect/cli/lib/run.js:577:17
┃     at Array.forEach (<anonymous>)
```

### Workaround

I have found that _not_ using `VerboseReporter` and instead adding the `--verbose` flag to the `asp` command enables logging successfully.

### Specifications
Node: 12.13.1
@as-pect/assembly: 3.1.1
@as-pect/cli: 3.1.1
@as-pect/core: 3.1.1
as-wasi: 0.0.1
assemblyscript: 0.9.4
ts-node: 8.7.0
typescript: 3.8.3

**Comment by jtenner** [comment]
Ah yeah. You can fix this by setting the reporter's stdout property like this:

```
reporter.stdout = process.stdout;
```

In the meantime, just using the --verbose flag is the intended way of consuming that reporter.

**Comment by willemneal** [comment]
fixed by #280

**Comment by jtenner** [comment]
Yep! There should already be a release out for this too. Fixed in `3.1.4`.
